src/main/target/NERO/target.h

```diff
@@ -136,4 +136,7 @@
 #define TARGET_IO_PORTD         (BIT(2))
 
 #define USABLE_TIMER_CHANNEL_COUNT 9
-#define USED_TIMERS             ( TIM_N(2) | TIM_N(3) | TIM_N(5) | TIM_N(8) | TIM_N(9) )
+#define PPM_PIN                 PC7
```

Comment:
I'd rather we did not reintroduce usage of these defines, considering they will have to be removed again when moving to universal targets.